R/pkg/R/serialize.R

```diff
@@ -83,6 +83,7 @@ writeObject <- function(con, object, writeType = TRUE) {
          Date = writeDate(con, object),
          POSIXlt = writeTime(con, object),
          POSIXct = writeTime(con, object),
+         bigint = writeDouble(con, object),
```

Comment:
When using createDataFrame, R uses `serialize` to send data to the backend. When taking an action, say, `collect`, scala side logic refers to the schema field and calls the `readTypedObjects` where the newly added read logic kicks in. When it returns back to R side, the newly added write logic kicks in and R side can interpret it due to the R side read logic. It seems that the `write` logic in R side is not called, because we don't have specific type `bigint` in R. Right?